web/content/js/nav.js

```diff
@@ -67,15 +61,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Planning Guide",
+		href: "/Planning_Guide/index-foreman-deb.html"
+	},{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
```

Comment:
I said it before, but I think we should get rid of the `on_Red_Hat` for this and proxy below. Just merge the two guides and use the flavors we have now.

Also, for me when I build this locally the page is empty?

Reply:
Agreed, created a ticket for @melcorr to arrange this downstream, I have a feeling that it might break downstream builds. Once we are all set, I can do the change: https://github.com/theforeman/foreman-documentation/issues/970